docs/cryptodoc/src/09_x509.rst

```diff
@@ -481,14 +479,16 @@ validation of the CertificatePolicies extension.
           ``issuer``'s public key, Append the corresponding error to
           ``cert_status[i]``
 
-      14. If ``issuer``'s public key strength < ``min_signature_algo_strength``
-          then do Append SIGNATURE_METHOD_TOO_WEAK to ``cert_status[i]``
+      14. If ``issuer``'s public key strength <
+          ``restrictions.minimum_key_strength()`` then do Append KEY_TOO_SMALL
+          to ``cert_status[i]`` then do Append SIGNATURE_METHOD_TOO_WEAK to
+          ``cert_status[i]``
```

Comment:
```suggestion
      14. If ``issuer``'s public key strength <
          ``restrictions.minimum_key_strength()`` then do Append SIGNATURE_METHOD_TOO_WEAK to
          ``cert_status[i]``
```

Reply:
Fair point! Thanks for the double-check.